### PR TITLE
CODE RUB: Updated change variable add Assignment Attachment

### PR DESCRIPTION
**Pull request by deepshri7**

### CODE REVIEW

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/AssignmentAttachments/AssignmentAttachmentServiceTests.Exceptions.Add.cs [hunk 1]
```diff
@@ -97,14 +97,13 @@ public async Task ShouldThrowServiceExceptionOnAddWhenExceptionOccursAndLogItAsy
             // given
             AssignmentAttachment randomAssignmentAttachment = CreateRandomAssignmentAttachment();
             AssignmentAttachment inputAssignmentAttachment = randomAssignmentAttachment;
-            var exception = new Exception();
-            var expectedAssignmentAttachmentServiceException = new AssignmentAttachmentServiceException(exception);
+            var Serviceexception = new Exception();
```
Comment (hassanhabib) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/AssignmentAttachments/AssignmentAttachmentServiceTests.Exceptions.Add.cs:100]: Remove the extra space at the end of this line.

Comment (hassanhabib) [reply OtripleS.Web.Api.Tests.Unit/Services/Foundations/AssignmentAttachments/AssignmentAttachmentServiceTests.Exceptions.Add.cs:100]: Also lower-case `ServiceException` to `serviceException`

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/AssignmentAttachments/AssignmentAttachmentServiceTests.Exceptions.Add.cs [hunk 2]
```diff
@@ -95,25 +95,21 @@ public async Task ShouldThrowDependencyExceptionOnAddWhenDbExceptionOccursAndLog
...
-            AssignmentAttachment someAssignmentAttachment = CreateRandomAssignmentAttachment();
-            var serviceException = new Exception();
-
-            var failedAssignmentAttachmentServiceException =
-                new FailedAssignmentAttachmentServiceException(serviceException);
-
-            var expectedAssignmentAttachmentServiceException =
-                new AssignmentAttachmentServiceException(
-                    failedAssignmentAttachmentServiceException);
+
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/AssignmentAttachments/AssignmentAttachmentServiceTests.Exceptions.Add.cs:98]: Remove extra line

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/AssignmentAttachments/AssignmentAttachmentServiceTests.Exceptions.Add.cs [hunk 3]
```diff
@@ -95,25 +95,21 @@ public async Task ShouldThrowDependencyExceptionOnAddWhenDbExceptionOccursAndLog
...
+            var serviceException = new Exception();   
+            var expectedAssignmentAttachmentServiceException = new AssignmentAttachmentServiceException(serviceException);
 
             this.storageBrokerMock.Setup(broker =>
-                broker.InsertAssignmentAttachmentAsync(someAssignmentAttachment))
+                broker.InsertAssignmentAttachmentAsync(inputAssignmentAttachment))
                     .ThrowsAsync(serviceException);
-
-            // when
+
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/AssignmentAttachments/AssignmentAttachmentServiceTests.Exceptions.Add.cs:107]: run code cleanup

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/StudentExamFees/StudentExamFeeServiceTests.Exceptions.RetrieveAll.cs [hunk 4]
```diff
@@ -24,14 +24,14 @@ public void ShouldThrowDependencyExceptionOnRetrieveAllStudentExamFeesWhenSqlExc
             this.storageBrokerMock.Setup(broker =>
                 broker.SelectAllStudentExamFees())
                     .Throws(sqlException);
-
+            
             // when
-            Action retrieveAllStudentExamFeesAction = () =>
-                this.studentExamFeeService.RetrieveAllStudentExamFees();
+            Action retrieveAllStudentExamFeeAction = () =>
+             this.studentExamFeeService.RetrieveAllStudentExamFees();
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/StudentExamFees/StudentExamFeeServiceTests.Exceptions.RetrieveAll.cs:30]: fix the tab here

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/StudentExamFees/StudentExamFeeServiceTests.Exceptions.RetrieveAll.cs [hunk 5]
```diff
@@ -58,17 +58,17 @@ public void ShouldThrowServiceExceptionOnRetrieveAllStudentExamFeesWhenException
 
             this.storageBrokerMock.Setup(broker =>
                 broker.SelectAllStudentExamFees())
-                    .Throws(serviceException);
-
+                  .Throws(exception);
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/StudentExamFees/StudentExamFeeServiceTests.Exceptions.RetrieveAll.cs:61]: Please run the build and test locally before committing the code.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

**Pull request closed without merging**